content/react.md

```diff
@@ -0,0 +1,271 @@
+---
+title: React
+order: 8
+description: How to use React, the frontend rendering library with Meteor.
+---
+
+After reading this guide, you'll know:
+
+1. What React is, and why you would consider using it with Meteor.
+2. How to install React in your Meteor application, and how to use it correctly.
+3. How to integrate React with Livedata, Meteor's realtime data layer.
+4. How to route in a React/Meteor application.
+
+<h2 id="introduction">Introduction</h2>
+
+[React](https://facebook.github.io/react/) is a JavaScript library for building reactive user interfaces developed and distributed by the Facebook team. React is one of the three rendering libraries supported by Meteor; it is an alterative to [Blaze](blaze.html) and [Angular](angular.html).
+
+React is has a vibrant and growing ecosystem and is used widely in production in a variety of combinations with different frameworks.
+
+To learn more about using React in general and coming up to speed with the library, you should check out the [React documentation](https://facebook.github.io/react/docs/getting-started.html), especially the [thinking in React](https://facebook.github.io/react/docs/thinking-in-react.html) post, which explains the React philosophy well.
+
+To get started with React in Meteor, you can follow along the [React tutorial](https://www.meteor.com/tutorials/react/creating-an-app). To see an example of a more complete Meteor application built with React, check out the [`react` branch](https://github.com/meteor/todos/tree/react) of the Todos exmaple application. Where applicable, code examples in this article will reference that app.
+
+<h3 id="using-with-meteor">Installing and using React</h3>
+
+To install React in Meteor 1.3 you should simply add it as an NPM dependency:
+
+```
+npm install --save react react-dom
+```
+
+This will install `react` into your project and allow you to access it within your files with `import React from 'react'`. Most React code is written in [JSX](https://facebook.github.io/react/docs/jsx-in-depth.html), which you can use by [default in Meteor](http://guide.meteor.com/build-tool.html#react-jsx) if you include the `ecmascript` package (which is installed in all Meteor apps by default).
+
+```js
+import React from 'react';
+
+export default class HelloWorld extends React.Component {
+  render() {
+    return <h1>Hello World</h1>;
+  }
+}
+```
+
+You can render a component heirarchy to the DOM using the `react-dom` package:
+
+```js
+import { Meteor } from 'meteor/meteor';
+import { render } from 'react-dom';
+import HelloWorld from './HelloWorld.jsx';
+
+Meteor.startup(() => {
+  render(HelloWorld, document.getElementById('app'));
+});
+```
+
+XXX: should we be getting rid of `blaze-html-templates` and replacing with `static-html`? The tutorial doesn't sem to do it..
```

Comment:
The tutorial doesn't do it because static-html didn't exist yet I think? And we end up using Blaze for accounts-ui anyway. So I'm not sure.